### ISSUE TITLE
Feature Request:  Track amount of damage done per unit/pilot

### ISSUE THREAD

**Issue by BLR-IIC** [MegaMek/megamek#1567]
In many of my games, kills are not everything.  I think it would be really neat if MegaMek was able to track the amount of damage each unit/pilot did in a game.  At a minimum I would like to see damage points done tracked, and maybe critical hits.  This could potentially translate to experience points automatically being distributed in MekHQ after a battle.

**Comment by neilfw** [comment]
I strongly agree.  I often use lighter faster mechs to finish off damaged mechs - lost legs, gyro damage, unconcious pilots whilst my heavier units go dealing damage.  The end of game "kill" allocation always seems out of balance.  Though pure damage has the disadvantage of smaller faster mechs (harder to hit you could say) have less armour so less damage to take.  Back in old-paper gaming days, I used to keep a matrix tally of damage dealt by each of my units against each opponent.
